shielder/deploy/deploy.sh

```diff
@@ -134,6 +134,14 @@ move_keys() {
 }
 
 docker_cargo() {
```

Comment:
what is this and why introduce this dichotomy?

Reply:
Explained in the comment.